src/back/routes/debug.ts

```diff
@@ -44,3 +48,14 @@ async function triggerFunction(req: WithAuth) {
     throw new Error(`Function '${functionName}' not found.`)
   }
 }
+function invalidateCache(req: WithAuth) {
+  const user = req.auth!
+  validateDebugAddress(user)
+
+  const { key } = req.body
+  if (!key || typeof key !== 'string') {
+    throw new Error(`Invalid cache key: ${key}`)
+  }
+
+  return CacheService.remove(key)
```

Comment:
What happens if we use an invalid key? Being invalid a key that we are not using in our backend.

Reply:
<img width="1063" alt="image" src="https://github.com/decentraland/governance/assets/2858950/f0fd1d52-5141-48ea-abca-f54dfadb0daf">

safu